client/layout/guided-tours/tours/simple-payments-email-tour.js

```diff
@@ -20,6 +20,15 @@ import {
 } from 'layout/guided-tours/config-elements';
 import { AddContentButton } from '../button-labels';
 
+// When moving from stats to the editor, the menu disappears, the first step
+// loses its target, and it repositions in the top left corner.
+// This function immediately moves it outside of the viewport, and, as soon
+// as the next step's target appears, the step is repositioned correctly.
+const handleTargetDisappear = () => {
+	const tourFirstStep = document.querySelector( '.guided-tours__step-first' );
+	tourFirstStep.style.left = '-9999px';
```

Comment:
I would recommend opacity hidden, we're only hiding this for a short time correct?

Reply:
That was my first thought, but all the steps are rendered as the same element, and just repositioned.
If I were to make a step invisible on "leave", I'd need to also make it back visible on next step "enter".
It might be possible by adding inline styles on the next step, but I'd say it's a bit more cumbersome.

Reply:
Got it, let's go with this approach then 👍